lms/static/sass/elements/_program-card.scss

```diff
@@ -108,52 +108,79 @@
     line-height: 1;
   }
 
-  .certificate-status {
-    margin-bottom: 0;
+  .status-text {
+    display: flex;
+    margin-bottom: 5px;
+  }
 
-    .status-text {
-      font-size: font-size(x-small);
-      color: palette(grayscale, dark);
-      line-height: 1;
-    }
+  .number-status {
+    text-align: center;
+    width: 100%;
+    float: left;
+    padding-left: 5px;
+    padding-right: 5px;
+    margin-top: -8px;
+    padding-bottom: 8px;
```

Comment:
You have 3 paddings defined so it would be easier to read if you used this syntax:
```
padding: {
    left: 5px;
    right: 5px;
    bottom: 8px;
}
```